pom.xml

```diff
@@ -631,7 +648,12 @@
       <dependency>
         <groupId>org.checkerframework</groupId>
         <artifactId>checker-qual</artifactId>
-        <version>3.0.0</version>
+        <version>3.8.0</version>
+      </dependency>
+      <dependency>
```

Comment:
Can you move this definition to a different section? you see that:
<!-- version resolution --> is in 592
<!-- styx client libs --> in 432

this should be on a section for api service